### ISSUE TITLE
[Question] Possible to remove Tehnut's Maven from build.gradle?

### ISSUE THREAD

**Issue by ALongStringOfNumbers** [GregTechCE/GregTech#1383]
It might be possible to remove Tehnut's maven from build.gradle. 

First off, the current location for his maven (`tehnut.info/maven`) is incorrect due to some issues he had with his maven a bit ago. The correct location has moved to `maven.tehnut.info`. Since we have not needed to correct this maven address since the change happened, we can assume that it might not be needed by the project.

Second, I am not sure what mods we would need from Tehnut's maven, as it looks like none of the current dependencies or run time includes would actually be from his maven. The only one I could think about would be for HWYLA, however, GTCE builds against TOP, and does not do anything for HWYLA, so I am unsure if the maven is even needed for the project a second time.

**Comment by Exaxxion** [comment]
Regarding this question, I commented out the tehnut maven definition in `build.gradle.kts`, ran the `clean` task, and for a proper tabula rasa I deleted my gradle cache directory.

I ran the following and the build succeeded without issue. The `setupDecompWorkspace` task is critical for the project to build, as otherwise there will be missing dependencies.

```bash
$./gradlew clean
$ rm -rf ~/.gradle/caches
$./gradlew setupDecompWorkspace
$./gradlew build
```

I think it's safe to say that we do not rely on tehnut's maven whatsoever.
